### ISSUE TITLE
Loading multiple postscribe calls on same page in parallel

### ISSUE THREAD

**Issue by neeagl** [krux/postscribe#135]
Hi,

We're trying to use postscribe in our script, however there's another script on the webpage which uses postscribe and there are run-time conflicts happening between the first & the second postscribe call.

Any help would be appreciated.

Thanks in advance

**Comment by jguerine** [comment]
Similar-ish issue here.
For some specific ad formats, our adserver requires to implement the adserver calls simulataneously and not one after each other.
Is there a way with postscribe to write all the placements in the div, delay until all the calls are prepared and then do the calls simultaneously?

Thanks !

**Comment by jnewman** [comment]
Can I get a place where I can reproduce this? @jguerine pls file a separate ticket I don't think your concern is related (but, I don't know much about either issue..)

**Comment by jnewman** [comment]
@neeagl For current version of postscribe, you can capture the value. The next version will support more ways to load. Capturing would look something like (assuming you have a build step):

```js
var myCopy = (function() {
  var oldPostscribe = window.postscribe;
  delete window.postscribe;
  // original postscribe code
  var myPostscribe = window.postscribe;
  window.postscribe = oldPostscribe;
  return myPostscribe;
})();
```

**Comment by sethyates** [comment]
We believe this is fixed in 2.0.0.  Please retest against 2.0.0 release and open a new issue following Issue Reporting guidelines in 2.0.0 if you believe this is still an issue.
